websites/P/PERPHeads/presence.ts

```diff
@@ -0,0 +1,182 @@
+const presence = new Presence({
+  clientId: "798312419260104705" // CLIENT ID FOR YOUR PRESENCE
+});
+
+let user: any, search: any, title: any;
+
+const browsingStamp = Math.floor(Date.now() / 1000);
+
```

Comment:
merge this with const from line 1

Reply:
not 100% on what you mean to be honest, sorry

Reply:
const a = "bla",
b = "foo";